aait/backend/group-2B/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
this kind of file should not be tracked. add to .gitignore

Reply:
Noted. All sensitive files have been gitignored.